packages/@aws-cdk-testing/cli-integ/tests/cli-integ-tests/cli.integtest.ts

```diff
@@ -775,6 +775,158 @@ integTest('cdk diff --fail with multiple stack exits with if any of the stacks c
+`;
+
+  expect(diff).toContain('Resource');
+  expect(diff).toContain('permission-set-with-managed-policy');
+
+  expect(diff).toContain('InstanceArn');
+  expect(diff).toContain('arn:aws:sso:::instance/testvalue');
+
+  expect(diff).toContain('PermissionSet name');
+  expect(diff).toContain('niceWork');
+
+  expect(diff).toContain('PermissionsBoundary');
+  expect(diff).toContain('ManagedPolicyArn: arn:aws:iam::aws:policy/AdministratorAccess');
+
+  expect(diff).toContain('CustomerManagedPolicyReferences');
+  expect(diff).toContain('Name: forSSO, Path:');
+}));
+
+integTest('cdk diff --security-only successfully outputs sso-assignment information', withDefaultFixture(async (fixture) => {
+  const diff = await fixture.cdk(
+    ['diff', '--security-only', fixture.fullStackName('sso-assignment')],
+  );
+  `┌───┬───────────────┬──────────────────────────────────┬─────────────────────────┬──────────────────────────────┬───────────────┬──────────────┬─────────────┐
+   │   │ Resource      │ InstanceArn                      │ PermissionSetArn        │ PrincipalId                  │ PrincipalType │ TargetId     │ TargetType  │
+   ├───┼───────────────┼──────────────────────────────────┼─────────────────────────┼──────────────────────────────┼───────────────┼──────────────┼─────────────┤
+   │ + │\${assignment} │ arn:aws:sso:::instance/testvalue │ arn:aws:sso:::testvalue │ 11111111-2222-3333-4444-test │ USER          │ 111111111111 │ AWS_ACCOUNT │
+   └───┴───────────────┴──────────────────────────────────┴─────────────────────────┴──────────────────────────────┴───────────────┴──────────────┴─────────────┘
+`;
+  expect(diff).toContain('Resource');
+  expect(diff).toContain('assignment');
+
+  expect(diff).toContain('InstanceArn');
+  expect(diff).toContain('arn:aws:sso:::instance/testvalue');
+
+  expect(diff).toContain('PermissionSetArn');
+  expect(diff).toContain('arn:aws:sso:::testvalue');
+
+  expect(diff).toContain('PrincipalId');
+  expect(diff).toContain('11111111-2222-3333-4444-test');
+
+  expect(diff).toContain('PrincipalType');
+  expect(diff).toContain('USER');
+
+  expect(diff).toContain('TargetId');
+  expect(diff).toContain('111111111111');
+
+  expect(diff).toContain('TargetType');
+  expect(diff).toContain('AWS_ACCOUNT');
+}));
+
+integTest('cdk diff --security-only successfully outputs sso-access-control information', withDefaultFixture(async (fixture) => {
+  const diff = await fixture.cdk(
+    ['diff', '--security-only', fixture.fullStackName('sso-access-control')],
+  );
+  `┌───┬────────────────────────────────┬────────────────────────┬─────────────────────────────────┐
+   │   │ Resource                       │ InstanceArn            │ AccessControlAttributes         │
+   ├───┼────────────────────────────────┼────────────────────────┼─────────────────────────────────┤
+   │ + │ \${instanceAccessControlConfig} │ arn:aws:test:testvalue │ Key: first, Values: [a]         │
```

Comment:
the pipe characters are off by a space on this line, could we fix that alignment?